account_move_report/__manifest__.py

```diff
@@ -9,10 +9,7 @@
         "account",
     ],
     "demo": [],
-    "data": [
-        "views/account_reports.xml",
-        "views/account_move_report.xml"
-    ],
+    "data": ["views/account_reports.xml", "views/account_move_report.xml"],
```

Comment:
Add a trailing comma so one item per line is preserved

Reply:
Done